index.html

```diff
@@ -20,7 +20,7 @@ <h2 class="greetings">Glad to meet you!</h2>
                 <p>If you will like what you see and have a project you will need coded,don't hesitate to contact me</p> 
                 <p>look through some of my work and experince</p>
             </div>
-            <h3 class="heading" style="color:blue;">LET'S CONNECT</h3>
+            <h3 class="heading">LET'S CONNECT</h3>
             <div class="icon-bar">
                 <img src="/home/ignatius/Documents/Microverse Projects/Portfolio/images/twitter.png" alt="">
                 <img src="/home/ignatius/Documents/Microverse Projects/Portfolio/images/linkedin.png" alt="">
```

Comment:
- [ ] [REQUIRED] Nice work but the list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag) and remove the `nav` tag because `nav` tag should create inside `header` tag for navigation elements. Please implement my suggestions 🙏🏼